### PR TITLE
893 - Enable to set LOGLEVEL by environment variable

### PR DESCRIPTION
**Pull request by jirik**
Part of issue #893

- ~~Tests~~
- ~~Layman Test Client (including docker image at docker hub)~~
- [x] Changelog
- [x] Documentation

### CODE REVIEW

File: .env.demo [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # Layman settings suitable for demo ##########################################
 ##############################################################################
 
+# logging level
+LAYMAN_LOGLEVEL=DEBUG
```
Comment (index-git) [inline .env.demo:6]: I expected `INFO` for demo, try it.

**Review by index-git — CHANGES_REQUESTED** [review]

**Review by index-git — APPROVED** [review]

File: .env.demo [hunk 2]
```diff
@@ -2,6 +2,9 @@
 # Layman settings suitable for demo ##########################################
 ##############################################################################
 
+# logging level
+LAYMAN_LOGLEVEL=INFO
```
Comment (jirik) [inline .env.demo:6]: I prefer to keep `DEBUG` for demonstration purposes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,7 @@
 ### Changes
 - [#880](https://github.com/LayerManager/layman/issues/880) Use Docker Compose v2 (`docker compose`) in Makefile without `compatibility` flag and remove `Makefile_docker-compose_v1` file. Docker containers are named according to Docker Compose v2 and may have different name after upgrade.
 - [#765](https://github.com/LayerManager/layman/issues/765) Stop saving OAuth2 claims in filesystem, use prime DB schema only.
+- [#893](https://github.com/LayerManager/layman/issues/893) It is possible to specify logging level by new environment variable [LAYMAN_LOGLEVEL](doc/env-settings.md#LAYMAN_LOGLEVEL). Default level is `DEBUG`. For production instances `INFO` is sufficient.
```
[the remark this answers is not in the document]
Comment (jirik) [reply CHANGELOG.md:14]: Anyway, default value is `DEBUG` if the variable is not set.

**Review by index-git — CHANGES_REQUESTED** [review]

File: doc/env-settings.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 ## General Layman settings
 
+### LAYMAN_LOGLEVEL
+Logging level. Default value is `DEBUG`, other possible values are `FATAL`, `ERROR`, `WARNING`, and `INFO`.
```
Comment (jirik) [inline doc/env-settings.md:6]: Default value is `INFO`

Comment (jirik) [inline doc/env-settings.md:6]:
```suggestion
Logging level. Default value is `INFO`, other possible values are `FATAL`, `ERROR`, `WARNING`, and `DEBUG`.
```

**Review by index-git — APPROVED** [review]

**Pull request merged**